factcast-test/src/main/java/org/factcast/test/FactcastConfig.java

```diff
@@ -0,0 +1,40 @@
+package org.factcast.test;
+
+import java.lang.annotation.*;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.With;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Inherited
+public @interface FactcastConfig {
+  String factcastVersion() default Config.FACTCAST_VERSION;
+
+  String postgresVersion() default Config.POSTGRES_VERSION;
+
+  String configDir() default Config.CONFIG_DIR;
+
+  @Value
+  @With
+  class Config {
+    String factcastVersion;
+    String postgresVersion;
+    String configDir;
+
+    static final String FACTCAST_VERSION = "0.3.9";
```

Comment:
should default to latest?

Reply:
Done and its your fault if we debug two hours in half a year :D